CHANGELOG.md

```diff
@@ -59,6 +59,11 @@ These methods are not intended for end users
 - `_run` => `run`
 - `_runAsync` => `runAsync`
 
+### Internal methods removed
+
+`numWorkers` method removed
+`nextWorker` method removed
```

Comment:
> ### Internal methods removed
>
> `numWorkers` method removed
> `nextWorker` method removed

Currently this is a lie?!
https://github.com/pioardi/poolifier/pull/172/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR44

The got deprecated getters and this is already in the changelog

Reply:
Oh I di not removed them? Mmmmm too tired, thanks to catch.
Since that are not covered by unit tests I was thinking to remove them.

What are your thoughts?

Reply:
My thoughts: get rid of them
We have a major 2.0 breaking release
Why keeping deprecated code when we remove it later on anyways :shrug:

Reply:
I Will remove them and adjust the changelog

Reply:
Done, please have a look